content/python/concepts/random-module/terms/uniform/uniform.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '.uniform()'
+Description: 'Returns a pseudo-random floating-point number between two given numbers.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Random'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.uniform()` method takes two numbers as arguments and returns a pseudo-random floating-point number between them, inclusive of the first value, and possibly inclusive of the second value, depending on rounding.
+
+## Syntax
+
+```pseudo
+random.uniform(value1,value2)
+```
+
+Where `value1` and `value2` are numbers bounding the choice of a random floating-point number.
+
+## Example
+
+The below example prints the output of `random.uniform()` picking random floating point numbers between 10 and 15, 100 and 150, -10 and 10, and 0.75 and 0.90.
```

Comment:
```suggestion
The following example prints the output of a random floating-point number between 10 and 15, 100 and 150, -10 and 10, and 0.75 and 0.90.
```

Reply:
This describes the codebyte example below this example. Maybe we can opt for just a codebyte example and remove this regular ex.